CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/ReplicatorConfigurationRequestHandler.swift

```diff
@@ -191,6 +192,14 @@ public class ReplicatorConfigurationRequestHandler {
             if let maxRetryWaitTime = maxRetryWaitTime, let maxRetryWaitTimeDouble = Double(maxRetryWaitTime) {
                 config.maxAttemptWaitTime = maxRetryWaitTimeDouble
             }
+            if auto_purge != nil {
+                if auto_purge == "enabled" {
```

Comment:
@euniceatcouchbase 
Since other platforms, we are checking after converting to lowercase. Should we need something like this for iOS?

`auto_purge. lowercaseString` == "enabled" ?